osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -149,6 +149,7 @@ private void pushWhenLoaded()
 
                 pushDebounce = Scheduler.AddDelayed(() =>
                 {
+                    pushDebounce = null;

```

Comment:
This might cause issues if done so early (this task is repeating). Can this instead be set when leaving the screen?